packages/react-components/package.json

```diff
@@ -65,10 +65,11 @@
     "@types/jest": "~26.0.22",
     "@types/json-stringify-safe": "^5.0.0",
     "@types/node": "^14.14.10",
+    "@types/react": "^16.9.49",
     "@types/react-aria-live": "^2.0.0",
     "@types/react-dom": "^16.9.8",
     "@types/react-linkify": "^1.0.0",
-    "@types/react": "^16.9.49",
+    "@types/styled-components": "^5.1.25",
```

Comment:
Is this needed?

Reply:
styled-components? Nope doesnt look like its being used anywhere